Source/SwiftLintFramework/Rules/VariableNameRule.swift

```diff
@@ -47,8 +48,17 @@ public struct VariableNameRule: ASTRule, ConfigurationProviderRule {
     )
 
     private func nameIsViolatingCase(name: String) -> Bool {
-        let firstCharacter = name.substringToIndex(name.startIndex.successor())
-        return firstCharacter.isUppercase() && !name.isUppercase()
+        let secondIndex = name.startIndex.successor()
+        let firstCharacter = name.substringToIndex(secondIndex)
+        if firstCharacter.isUppercase() {
+            if name.unicodeScalars.count > 1 {
```

Comment:
It may not be problem but `name.characters.count` would be correct, because `String.Index` is `String.CharacterView.Index`.